serverlessworkflow/modules/ROOT/nav.adoc

```diff
@@ -107,6 +107,7 @@
 **** xref:use-cases/advanced-developer-use-cases/deployments/deploying-on-minikube.adoc[Deploying on Minikube]
 **** xref:use-cases/advanced-developer-use-cases/deployments/deploying-on-kubernetes.adoc[Deploying on Kubernetes]
 **** xref:use-cases/advanced-developer-use-cases/deployments/deploying-on-openshift.adoc[Deploying on OpenShift]
+**** xref:use-cases/advanced-developer-use-cases/deployments/deploying-sonataflow-management-console-on-openshift.adoc[Deploying Management Console on OpenShift]
```

Comment:
Please move this guide to `Tooling` section, or to `Cloud` not sure what is best, but it should not be in Quarkus advacned section

Reply:
Please, Cloud. Do not add under operator, thou.